vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflow.java

```diff
@@ -83,7 +70,7 @@ public void recover(String workflowName, String workflowVersion, String vidarrId
     final var stateWaiting = workflows.computeIfAbsent(workflowName, k -> new WaitingState()).waiting;
     // since we just created it if it doesn't exist, no need for null check here
 
-    int workflowPriority = 1;
+    int workflowPriority = (int) acceptedPriorities.get(0);
```

Comment:
rather than casting from Object to int, fix the 'raw use of parameterized class' on line 26 and use List<Integer>